cronjobs/bsz_util.py

```diff
@@ -82,18 +94,18 @@ def GetBackupDirectoryPath(config):
 
 # Check whether all the instances are needed
 def NeedsBothInstances(filename_regex):
-     without_localdata_pattern = "_o[)]?-[(]?.*[)]?"
-     without_localdata_regex = re.compile(without_localdata_pattern)
-     return without_localdata_regex.search(filename_regex.pattern) is not None
+    without_localdata_pattern = "_o[)]?-[(]?.*[)]?"
```

Comment:
General comment: It would be nice to use type hinting wherever possible, of course taking into account what the version of Python 3 on CentOS supports.